docs/docs/examples/guides/isaaclab-aica-bridge.md

```diff
@@ -127,13 +127,9 @@ To set up the hardware interface, follow these steps:
 1. Go to the **Hardware** tab in **AICA Studio**.
 2. Click on the Universal Robot 5e URDF to open it.
 3. Click **Save As** to create a copy of the URDF file for the UR5e robot in **AICA Studio**.
-4. In the **hardware** tag, replace the existing plugin with the lightweight interface plugin shown above.
-5. Remove any unused tags, such as GPIO or other unnecessary hardware interfaces, from the URDF file.
-6. Save your changes.
-7. In the **Hardware Interface** component on the graph view, select the modified URDF file for the UR5e robot from the
-   drop down menu in the hardware interface.
+4. In the **hardware** tag, replace the existing plugin with the lightweight interface plugin shown below.
+
 
-```xml
```

Comment:
we need this back in. Sorry, my bad to an extent, I just couldn't comment a suggestion and also keep this in 😋